pkg/replicator/replicater.go

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package replicator
```

Comment:
I think go best practice would rather be "replicate".

Reply:
ping @bwplotka @GiedriusS

Reply:
Yes, I agree. Sorry @daixiang0 for initially asking you to rename to `replicator`. `replicate` makes sense as per:  https://thanos.io/contributing/#components-naming